02-oop-inheritance/Exercise 8 finished/js/Movies.js

```diff
@@ -0,0 +1,28 @@
+import MyClassEventEmitter from "EventEmitter";
+
+export default class Movie extends MyClassEventEmitter {
+
+    constructor(title, year, duration) {
+        super();
+        this.title = title;
+        this.year = year;
+        this.duration = duration;
+        this.ActorArray = [];
+    }
+
+    play() {
+        EventEmitterObj.emit("play");
```

Comment:
`EventEmitterObj`? Should be doing the reference to the father's contructor using `super`.